docs/installation.md

```diff
@@ -58,6 +58,13 @@ Most systems already have `gcc` installed, with the exception of "slim" Docker i
 1. Use `python:3.11-buster` instead of `python:3.11-slim-buster`, which includes gcc.
 1. Run `apt update && apt install build-essential -y` (e.g. in your Dockerfile) to install gcc.
 
+### 3. The error message `Failed to build sudachipy`
+
+The `sudachipy` package, which is installed with `langcheck[all]` or `langcheck[ja-optional]`, does not have a pre-built wheel for ARM64 Linux devices.
```

Comment:
Maybe we should also suggest installing each lang individually, i.e. `langcheck[en,ja,de,zh]`?

Reply:
Added the lines